src/commands/helm-lint.yaml

```diff
@@ -1,5 +1,25 @@
 ---
+parameters:
+  disable:
+    description: "Disable linting alogether"
+    type: "boolean"
+    default: false
+  lenient:
+    description: "Use more lenient lint rules"
+    type: "boolean"
+    default: false
 description: Lint Helm charts
 steps:
-  - run: |
-      ct lint --validate-maintainers=false --check-version-increment=false --chart-dirs helm
+  - unless:
+      condition: <<parameters.disable>>
```

Comment:
BTW I don't think it's going to work because condition works with a string where any non-empty string is true. At least that what I found in the docs back then. But if it works then great.

Reply:
Yeh, I think your right... Just saw boolean type and just presumed it would work. Just double-checked the docs it references strings only.

Will fix